lib/testLauncher/commands/interactive.js

```diff
@@ -44,29 +44,30 @@ const builder = yargs => {
 		.option('inspect', {
 			describe: 'Will launch user command with --inspect execArgv, used for debugging',
 			global: false,
+			type: 'number',
```

Comment:
is it work if just pass '--inspect' without port?

Reply:
oh, so it also has to work without it? What should happen then, webStorm will pick any available port?

Reply:
it should work this way not only with webstorm but in other cases as well, it is how node works https://nodejs.org/api/debugger.html#debugger_v8_inspector_integration_for_node_js

Reply:
ok, cool, I'll change

Reply:
Actually good notice, in our json schema validators, type inspect and inspectBrk are set to numbers, so error is thrown if used without.

Reply:
Another thing is that if I run script as `suitest interactive --inspect mocha ...` ownArgs.inspect is set to string 'mocha'. Maybe to consider it as true you have to set type to boolen here. I think we need to clarify this and fix as separate issue. Maybe we should force user to set some value or

Reply:
@givehug could you please document it in a new Jira ticket, please? We'll take it into one of the following sprints.

Reply:
https://mineus.atlassian.net/browse/SUIT-12173